src/unum/devtelemetry/dt_collector.c

```diff
@@ -29,9 +29,17 @@ static void ip_pkt_rcv_cb(TPCAP_IF_t *tpif,
 // All (from or to unicast MAC) IP packets processing entry for
 // device & connection tracking
 static PKT_PROC_ENTRY_t collector_all_ip = {
+#ifdef FEATURE_FESTATS_ONLY
```

Comment:
Please try to remove the clutter and make it more readable by creating a separate data structure (for example "collector_stats_only") and conditionally use it instead of "collector_all_ip" in the tpcap call.  Please focus on readability and reducing the use of ifdefs when doing this (for example you might need to keep pe = collector_all_ip assignment and just override it with a single line pe = collector_stats_only under #ifdef FEATURE_FESTATS_ONLY, see if that works out to be cleaner and more readable).

Reply:
Handled it in a slightly different way. Without changing the name for each #ifdef I have created the structure with the same name but with matching flags as 0.